Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -464,6 +478,31 @@ await _cdfSimulatorResources.UpdateSimulatorModelRevisionData(
             }
         }
 
+        /// <summary>
+        /// Creates a new temporary memory cache to store model revisions.
+        /// We can use this cache to avoid fetching the same model revisions multiple times.
+        /// This only works if the model revisions do not have dependencies.
+        /// Dependencies field is not being returned by the list endpoint, so we need to call get-by-id instead of using cache.
+        /// </summary>
+        /// <param name="modelRevisions">List of model revisions to cache</param>
+        /// <returns>Memory cache with model revisions. Null if caching is not possible</returns>
+        private MemoryCache CreateMemoryCache(List<SimulatorModelRevision> modelRevisions)
+        {
+            MemoryCache cache = null;
+            var canUseCachedRevisions = !(_simulatorDefinition.ModelDependencies?.Any() == true);
```

Comment:
Can the user disable cache completely, as a extra safety guard?
Or is that also achieved by restarting the process.

Reply:
If there are many revisions, and this cache is disabled, the connector would not be able to handle them without this cache due to request limits.

Reply:
I see, as it will be short-lived there is less chance of permanent trouble due to potential invalid cache entries.